src/simulator/views/map/display/solid_colour_map_display_pot.py

```diff
@@ -26,12 +27,11 @@ def __init__(self, services: Services, points, z_index=50, radius=0, custom_map:
         self.small = min(pmaplst)
         self.range = (self.large-self.small)/6
         self.pts = set()
+        self.__point_dim = 3
```

Comment:
does this break 2d?

Reply:
I'll change that to None, because this value has no effect whatsoever.

Reply:
oh lol what can you just get rid of it entirely then?

Reply:
No because we still want (& I think need) the symbol to be defined